packages/tools/storybook/stories/cdk/overlay/examples/basic/overlay.basic.stories.tsx

```diff
@@ -54,6 +52,14 @@ const storyParams = {
     control: { type: 'select' },
     table: { defaultValue: { summary: 'ltr' } }
   },
+  changePosition: {
+    category: 'General',
+    description: 'position component on the page',
+    defaultValue: 'center',
+    options: ['left', 'right', 'center'],
```

Comment:
I know it's not on the story, but it would be nice to have `top` and `bottom` also (and `center` as middle)